README.md

```diff
@@ -1,42 +1,71 @@
-# StudyU built at HPI with ❤
+# StudyU: N-of-1 Trials Made Easy
 
-We have different Flutter/Dart packages all contained in this monorepo.
+<img src="https://www.studyu.health/img/icon_wide.png" height="130" align="right"
+alt="StudyU Icon">
 
-Active packages:
+StudyU is an <a href="https://www.studyu.health/blog/best-idea-award">
+award-winning</a> platform to discover the effects of individual health
```

Comment:
Is there a reason this can't be a markdown link?
```suggestion
StudyU is an [award-winning](https://www.studyu.health/blog/best-idea-award)
platform to discover the effects of individual health
```